romancal/regtest/scripts/create_jump_data.py

```diff
@@ -0,0 +1,148 @@
+import pickle
```

Comment:
I am not sure where to put this file, its just what I wrote in order to create all the regression test data. I thought it might want to within `romancal` so that if they need to be regenerated someone else would be able to do it.